verifier/verifier.go

```diff
@@ -518,6 +549,76 @@ func verifyAuthenticTimestamp(outcome *notation.VerificationOutcome) *notation.V
 	}
 }
 
+func verifyRevocation(outcome *notation.VerificationOutcome, r revocation.Revocation, logger log.Logger) *notation.ValidationResult {
+	if r == nil {
+		return &notation.ValidationResult{
+			Type:   trustpolicy.TypeRevocation,
+			Action: outcome.VerificationLevel.Enforcement[trustpolicy.TypeRevocation],
+			Error:  fmt.Errorf("unable to check revocation status, revocation client cannot be nil"),
+		}
+	}
+
+	authenticSigningTime, err := outcome.EnvelopeContent.SignerInfo.GetAuthenticSigningTime()
+	if err != nil {
+		logger.Debugf("not using authentic signing time due to error retrieving AuthenticSigningTime, err: %v", err)
+	}
+	revocationResult, err := r.Validate(outcome.EnvelopeContent.SignerInfo.CertificateChain, authenticSigningTime)
```

Comment:
nit:
can we rename `revocationResult` to `revocationResults` to reflect it's an array?
```suggestion
	revocationResults, err := r.Validate(outcome.EnvelopeContent.SignerInfo.CertificateChain, authenticSigningTime)
```